packages/tictactoe/src/redux/__redux-bookmarks__/notes.md

```diff
@@ -0,0 +1,4 @@
+# Redux bookmarks
```

Comment:
@geoknee I like this idea, but would other developers be able to reuse these bookmarks? Does redux state contain addresses, requiring your browser + Metamask to hold the correct private keys to sign states + make transactions?

Reply:
Yes, I think you're right about this (so have removed). Still a useful workflow, I think, but probably requires each dev to create these bookmarks themselves (once, and not to check them in).

Reply:
Agreed, I use saved redux state frequently.